pkg/lib/cockpit-components-listing-panel.scss

```diff
@@ -36,6 +39,7 @@
         }
       }
     }
+    }
```

Comment:
That looks like a wrong indentation somewhere?

Reply:
not present anymore.